src/shaders/collision_box.vertex.glsl

```diff
@@ -14,14 +14,15 @@ varying float v_placed;
 varying float v_notUsed;
 
 void main() {
-    vec4 projectedPoint = u_matrix * vec4(a_anchor_pos, elevation(a_anchor_pos), 1);
+    vec4 projectedPoint = u_matrix * vec4(a_pos + elevationVector(a_anchor_pos) * elevation(a_anchor_pos), 1);
```

Comment:
This needs to be reviewed: in the original implementation we calculated the `projectedPoint` using  `a_anchor_pos`

Reply:
This should be fine. `a_pos` has been promoted to a 3-component vector in order to support arbitrary direction vectors. If you have a look at the function `_addCollisionDebugVertices` in symbol_bucket.js and follow the call chain you'll notice that the `a_pos` is populated with projected anchor value. In non-globe view this equals to tile anchor where z is 0.

Naming of the anchor variables might not be the best possible if you follow the call chain in symbol bucket. :)